scripts-dev/release.py

```diff
@@ -295,6 +295,13 @@ def tag(gh_token: Optional[str]):
     if click.confirm("Edit text?", default=False):
         changes = click.edit(changes, require_save=False)
 
+    commit = repo.head.commit
+    click.echo(
+        f"{repo.head.ref} {commit} {commit.summary!r}\n"
+        f"by {commit.author} <{commit.author.email}>, "
+        f"committed at {commit.committed_datetime}"
+    )
+    click.confirm(f"Tag this commit as {tag_name}?", default=False, abort=True)
```

Comment:
I'd be tempted to fail if we're not on the right branch, maybe? We already check the right branch in another command:

https://github.com/matrix-org/synapse/blob/706456de1f63a4b9796592fd452da7ce30735274/scripts-dev/release.py

Reply:
Do you have a line number for me, sorry? I think that link is missing one